### PR TITLE
Fix UG after Peer Review

### LINKED ISSUE #342: Peer Review: No description of symbol meanings
e.g. Note, Alert

### LINKED ISSUE #344: UG Peer Review: Placeholder restrictions contradict invalid examples

### LINKED ISSUE #345: UG Peer Review: Inconsistent formatting in FoodRem commands
e.g. Format: dec INDEX vs Command: rmk ITEM_INDEX

### PR DESCRIPTION
**Pull request by Ferusel**
Fixes #345 
Fixes #344 
Fixes #342

### CODE REVIEW

File: docs/_ug/HowToUseUserGuide.md [hunk 1]
```diff
@@ -20,3 +20,13 @@ If you are an experienced user, you can refer to the [Command Summary](#command-
 If you are stuck, refer to the section on [Troubleshooting](#troubleshooting) or [FAQ](#faq).
 
 You can also refer to the [Glossary](#glossary) for definitions of commonly used terms in FoodRem.
+
```
Comment (RichDom2185) [inline docs/_ug/HowToUseUserGuide.md:23]: Should we put a

```markdown
#### Admonishments
<!-- or maybe ##### -->

Admonishments are...
```

here?

File: docs/_ug/Installation.md [hunk 2]
```diff
@@ -9,8 +9,12 @@ If you haven't installed FoodRem, simply follow these steps to set it up:
 1. Copy the jar file to an empty folder. This will be your home folder for FoodRem.
 1. Double-click on the jar file to launch FoodRem.
 
-```note 
-If FoodRem does not launch after double-clicking the jar file, check out the [Troubleshooting](#troubleshooting) section for an alternative solution.
+```warning
+Remember to add `foodrem.jar` into an empty folder as additional files will be created when launching FoodRem for the first time!
```
Comment (RichDom2185) [inline docs/_ug/Installation.md:13]: Might be good to explain "additional files", e.g. "additional data and configuration files"

File: docs/_ug/Installation.md [hunk 3]
```diff
@@ -9,8 +9,12 @@ If you haven't installed FoodRem, simply follow these steps to set it up:
...
-If FoodRem does not launch after double-clicking the jar file, check out the [Troubleshooting](#troubleshooting) section for an alternative solution.
+```warning
+Remember to add `foodrem.jar` into an empty folder as additional files will be created when launching FoodRem for the first time! 
 ```
 
 Congratulations! You now have FoodRem set up.
+
+```danger
+When you launch FoodRem, notice that you will have a folder called "data" that is created in the same folder as `foodrem.jar`. Editing the file might corrupt your data, avoid doing this!
+```
```
Comment (RichDom2185) [inline docs/_ug/Installation.md:20]: Nice catch!

File: docs/_ug/KeyDefinitions.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ The following are the attributes stored for each item:
 * Item remarks
 * Item tags
 
-FoodRem Items are unique by name and case-sensitive. This means you cannot add two or more items of the same name.
+FoodRem Items are unique by name and case-insensitive. This means you cannot add two or more items of the same name.
```
Comment (RichDom2185) [inline docs/_ug/KeyDefinitions.md:17]: They are case sensitive. "Potato" and "potato" are different. Right? @Eugenetanwl3881

File: docs/_ug/KeyDefinitions.md [hunk 5]
```diff
@@ -32,7 +32,7 @@ Feel free to add tags as you see fit to organize your inventory. Examples of how
 
 Tags can be renamed and these changes would be reflected on all items immediately.
 
-FoodRem Tags are unique by name and case-sensitive. This means you cannot add two or more tags of the same name.
+FoodRem Tags are unique by name and case-insensitive. This means you cannot add two or more tags of the same name.
```
Comment (RichDom2185) [inline docs/_ug/KeyDefinitions.md:35]: They are case sensitive. "Potato" and "potato" are different. Right? @Eugenetanwl3881

**Review by RichDom2185 — APPROVED** [review]
LGTM, just some comments below:

File: docs/_ug/KeyDefinitions.md (hunk 4 above)
Comment (Eugenetanwl3881) [reply docs/_ug/KeyDefinitions.md:17]: Right now, yes

File: docs/_ug/KeyDefinitions.md (hunk 5 above)
Comment (Eugenetanwl3881) [reply docs/_ug/KeyDefinitions.md:35]: Right now, yes. Unless we planning to change

File: docs/_ug/HowToUseUserGuide.md (hunk 1 above)
Comment (Ferusel) [reply docs/_ug/HowToUseUserGuide.md:23]: Good point, will add it

Comment (Ferusel) [reply docs/_ug/HowToUseUserGuide.md:23]: I will replace Admonishments with FoodRem Icons, as Admonishments might be too technical

**Review by RichDom2185 — APPROVED** [review]
LGTM, thanks a bunch!

**Pull request merged**